clarity_ext/domain/container.py

```diff
@@ -81,26 +81,33 @@ class Container(DomainObjectMixin):
     CONTAINER_TYPE_96_WELLS_PLATE = 100
     CONTAINER_TYPE_STRIP_TUBE = 200
     CONTAINER_TYPE_TUBE = 300
+    CONTAINER_TYPE_PATTERNED_FLOW_CELL = 400
```

Comment:
Yes, these are as you say IDs. The `size` mentioned in the commit log refers to the size of the container in the number of wells it contains (rows and columns.)